delegated-events.js.flow

```diff
@@ -0,0 +1,13 @@
+/* @flow */
+
+type EventHandler = (event: Event) => mixed
+
+type EventListenerOptions = {
```

Comment:
For delegated-events, I never implemented boolean support.

https://github.com/dgraham/delegated-events/blob/74177cedd0cf69d6b93fdb9bd89c3fe3d5fa4fb7/delegated-events.js#L80